### DISCUSSION TITLE
GitHub Copilot Stop working for certain words

### DISCUSSION THREAD

**Discussion by m-sarabi** [community/community#15494] [Copilot Conversations]
I'm writing candlestick pattern recognition with python and GitHub Copilot does work perfectly, but after this comment: `# Hanging Man` It just stop working. If I write anything else or censor it like `# Hang-ing M-an` It works again. I wonder if it is hardcoded for certain words or a bug!

**Comment by ctx400** [comment]
I've seen this too. It seems like any "non-PC" word that anyone might find offensive in your source causes copilot to refuse to generate suggestions. I've had it before where a "censor word" is completely innocent in the context used in my source, but its mere presence in the source means copilot will refuse to run.

**Reply by ctx400** [reply]
@github-staff Could anyone at GitHub confirm this is the case? Is there a word blocklist that developers can't use when generating code with copilot?

**Comment by shouya** [comment]
I have found two other words when taking my notes on a course about schizoprenia: "suicide" and "[substantia nigra](https://en.wikipedia.org/wiki/Substantia_nigra)". It's hilarious that copilot stops working on such low-level triggers. Putting dashes between the letters could get around it.

**Comment by gusbemacbe** [comment]
@m-sarabi, @ctx400 and @shouya 

Putting dashes between the letters will not work because Copilot reads the whole text and refuses to generate the code even if you put the dashes between the letters.

I suggest you try CodeGeexX. It does not censor the terms, as I tested.
